### PR TITLE
Document SMPP status events

### PR DESCRIPTION
**Pull request by justinvdm**
#983-#992 added status events, we need to document them.

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

File: docs/transports/smpp.rst [hunk 1]
```diff
@@ -112,3 +112,129 @@ message_id in the response.
...
+  - ``component``: ``smpp``
+
+
+``connection_lost``
+~~~~~~~~~~~~~~~~~~
+
+Published when a transport loses its connection to the SMSC. This occurs in the
+following situations:
+
+  - after succesfully unbinding
```
Comment (rudigiesler) [inline docs/transports/smpp.rst]: s/succesfully/successfully

**Comment by rudigiesler** [comment]
Just the one minor comment. Looks good otherwise.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**